cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,93 @@
+/* eslint-disable cypress/unsafe-to-chain-command */
+/* eslint-disable arrow-parens */
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+
+  it('should have pagination', () => {
+    // pagination
+    cy.get('.-pageInfo').should('contain', 'Page');
+    cy.get('.-pageJump > input').type('Number');
+    cy.get('.-totalPages').should('contain', '1');
+    cy.get('.-btn').should('contain', 'Previous');
+    cy.get('.-btn').should('contain', 'Next');
+  });
+
+  it('should have rows count selection', () => {
+    // Rows count selection
+    cy.get('select').should('contain', '5 rows');
+    cy.get('select').should('contain', '10 rows');
+    cy.get('select').should('contain', '20 rows');
+    cy.get('select').should('contain', '25 rows');
+    cy.get('select').should('contain', '50 rows');
+    cy.get('select').should('contain', '100 rows');
+  });
+
+  it('should allow to add a new worker', () => {
+    cy.get('#addNewRecordButton').should('contain', 'Add').click();
+    cy.get('#registration-form-modal').should('contain', 'Registration Form');
+    cy.findByPlaceholder('First Name').type(user.firstName);
+    cy.findByPlaceholder('Last Name').type(user.lastName);
+    cy.findByPlaceholder('name@example.com').type(user.email);
+    cy.findByPlaceholder('Age').type(user.age);
+    cy.findByPlaceholder('Salary').type(user.salary);
+    cy.findByPlaceholder('Department').type(user.department);
+    cy.get('#submit').should('contain', 'Submit').click();
```

Comment:
the same here. Should() is not necessary

Reply:
I've got your idea. That is true. Thank you.